proto/sample_v2_round_trip.json

```diff
@@ -4991,4 +4991,4 @@
         "total": 12345.6789
     },
     "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3358.0 Safari/537.36"
-}
```

Comment:
I wonder why this happened?  Unix vs. windows proto diff maybe?

Reply:
some people have editor settings for this, though maybe you'd have to be committing through your editor for those to be triggered for these files?